### PR TITLE
Add to block and file storage volume list order.

### PR DESCRIPTION
**Pull request by FernandoOjeda**
Add to block and file storage volume list order https://github.com/softlayer/softlayer-python/issues/1339.

### CODE REVIEW

File: SoftLayer/managers/block.py [hunk 1]
```diff
@@ -25,9 +25,10 @@ def list_block_volume_limit(self):
         """
         return self.get_volume_count_limits()
 
-    def list_block_volumes(self, datacenter=None, username=None, storage_type=None, **kwargs):
+    def list_block_volumes(self, datacenter=None, username=None, order=None, storage_type=None, **kwargs):
```
Comment (allmightyspiff) [inline SoftLayer/managers/block.py:28]: the new parameter `order` should go after `storage_type`, not before. This way anyone passing in arguments without the names wont encounter an error.

File: SoftLayer/managers/file.py [hunk 2]
```diff
@@ -22,9 +22,10 @@ def list_file_volume_limit(self):
         """
         return self.get_volume_count_limits()
 
-    def list_file_volumes(self, datacenter=None, username=None, storage_type=None, **kwargs):
+    def list_file_volumes(self, datacenter=None, username=None, order=None, storage_type=None, **kwargs):
```
Comment (allmightyspiff) [inline SoftLayer/managers/file.py:25]: the new parameter order should go after storage_type, not before. This way anyone passing in arguments without the names wont encounter an error.

File: tests/CLI/modules/block_tests.py [hunk 3]
```diff
@@ -141,6 +141,27 @@ def test_volume_list(self):
             }],
             json.loads(result.output))
 
+    def test_volume_list_order(self):
+        result = self.run_command(['block', 'volume-list', '--order=1234567'])
+
+        self.assert_no_fail(result)
+        self.assertEqual([
```
Comment (allmightyspiff) [inline tests/CLI/modules/block_tests.py:148]: I dont like tests like this, they break too easily if the fixture is changed. 

instead just do 

```python
json_result = json.loads(result.output)
self.assertEqual(json_result['id'], 100)
```

or something. If a few of the fields you expect to be there are correct, likely they rest are as well.

File: tests/CLI/modules/file_tests.py [hunk 4]
```diff
@@ -55,6 +55,26 @@ def test_volume_list(self):
             }],
             json.loads(result.output))
 
+    def test_volume_list_order(self):
+        result = self.run_command(['file', 'volume-list', '--order=1234567'])
+
+        self.assert_no_fail(result)
+        self.assertEqual([
```
Comment (allmightyspiff) [inline tests/CLI/modules/file_tests.py:62]: I dont like tests like this, they break too easily if the fixture is changed.

instead just do

json_result = json.loads(result.output)
self.assertEqual(json_result['id'], 100)
or something. If a few of the fields you expect to be there are correct, likely they rest are as well.

**Review by allmightyspiff — CHANGES_REQUESTED** [review]
One thing missing is i'd like the output of a storage order to mention that `slcli block volume-list -o <orderid>` can be used to look up this order, like the `ibmcloud` command has.

Thanks.

File: SoftLayer/managers/block.py (hunk 1 above)
Comment (FernandoOjeda) [reply SoftLayer/managers/block.py]: Done.

File: SoftLayer/managers/file.py (hunk 2 above)
Comment (FernandoOjeda) [reply SoftLayer/managers/file.py]: Done.

File: tests/CLI/modules/block_tests.py (hunk 3 above)
Comment (FernandoOjeda) [reply tests/CLI/modules/block_tests.py]: Done.

File: tests/CLI/modules/file_tests.py (hunk 4 above)
Comment (FernandoOjeda) [reply tests/CLI/modules/file_tests.py]: Done.

**Review by allmightyspiff — APPROVED** [review]

**Comment by allmightyspiff** [comment]
## TESTING
- ✔️  Command shown when ordering a volume to check order status
- ✔️ Searching by order id works

**Pull request merged**